resources/assets/js/src/components/aerialPhoto/AerialPhoto.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import {Panel} from "react-bootstrap";
+import WindDirection from "../windDirection/WindDirection";
+
+const AerialPhoto = (props) => {
+    return (
+        <Panel className="current-photo">
+            <Panel.Heading>
+                <Panel.Title>Aerial photo</Panel.Title>
+            </Panel.Heading>
+            <Panel.Body>
+                <WindDirection direction={props.direction} speed={props.speed} suffix={props.suffix} />
+                <img src="../../../../images/map-google.JPG" alt="Aerial Photo" className="img-responsive" />
```

Comment:
It is possible to set images folder path as constant and append only the name of the file? Nesting too deep is not readable easily.